### ISSUE TITLE
A11y: MUST MAY, MAYBE SHOULDN'T?

### ISSUE THREAD

**Issue by bduga** [w3c/epub-specs#903]
In section 4.4.1.3: "must meet the following requirements" 

The list is introduced with a MUST, but has a MAY. MUST MAY is a bad conformance requirement. Also, repetition of the keywords is confusing. Suggest either dropping the "MUST" from the body text, or move the MAY requirement to a new list of one item, introduced by "In addition, it MAY include the following". I think dropping the MUST is cleaner - "A conformant EPUB Publication has the following requirements". Although still odd - a MAY is explicitly not a requirement. Can it just be dropped?

**Comment by mattgarrish** [comment]
It should say "following criteria" to match the core specs (which have this pattern used all over).

If I remember right, this phrasing was used to avoid potentially having three lists of conformance requirements for every section -- one for each level of normative keyword. The first must is effectively only establishing that each criteria is applicable, not that the requirement defined in each must be met.

**Comment by mattgarrish** [comment]
I haven't been able to think of better alternative wording for this issue. If we take out the must from the lead-in sentence, it sounds like we forgot to note that adherence is normative. I'd argue to leave this alone for this revision, as it's late in the process and if we make a change here we really should be changing all the content and reading system conformance sections across all the specifications.

The reason the page markers need mentioning is that it causes confusion as we don't detail the usage in the core specifications. In an epubcheck discussion a while back someone thought that page lists and page markers were mutually exclusive and that if you picked one that was enough. Even though it's optional to include, it's useful for clarity. I don't want to peel it away from the other requirements, as that starts down the path to separate must/should/may groupings.

And for the record, I'm not trying to argue that the current approach is perfect, only that it works well enough for its purpose.

**Comment by bduga** [comment]
I can live with leaving it. I think I complained about the same thing in the other specs as well. I just dislike saying you MUST MAY do something.

**Comment by mattgarrish** [comment]
I'd like to change it up, but my thoughts are more radical, like pulling out reading system conformance to a dedicated document and maybe rethinking the need for so many content conformance sections (most are just pointers to where there are actual requirements). Definitely not for this revision, though.
